qemu/tests/cfg/vfio_net_boot.cfg

```diff
@@ -28,6 +28,15 @@
     variants ip_version:
         - ipv4:
         - ipv6:
+    variants:
+        - vfio:
```

Comment:
Hum, how about `@default` or `without_iommufd`, the `@` operator will hide the variant name. This case is already for **VFIO** and does not need to be declared again.

Reply:
@PaulYuuu  Updated, and thank you for explaining what @ mean!